src/lib/api-client.js

```diff
@@ -37,14 +38,31 @@ function post(endpoint, data) {
     });
 }
 
+export function checkInItem(studentId, itemAddress){
+    post('checkin', {
+        studentId,
+        itemAddress
+    }).then(data => {
+        Dispatcher.handleAction('CHECKIN_SUCCESS', {
+            itemAddress: data.itemAddress
+        });
+    }).catch(data => {
+        Dispatcher.handleAction('ERROR', {
```

Comment:
I missed one here. Same thing as the last comment, this should be an `error` string, not a `data` object.